index.html

```diff
@@ -0,0 +1,40 @@
+---
+layout: default
+main_css_classname: homepage
+---
+
+<div class="home">
+
+    <ul class="post-link-list list-unstyled">
+        {% for post in site.posts %}
+        {% assign author = site.data.authors[post.author] %}
+
+        <li class="post-link-item">
+            <a class="post-link" href="{{ post.url | prepend: site.baseurl }}">
+                <h2 class="post-link-item--title">{{ post.title }}</h2>
+            </a>
+
+            <span class="post-link-item--meta">Publié le {{ post.date | date: '%d' }}
+                {% assign m = post.date | date: "%-m" %}
+                {% case m %}
```

Comment:
est-ce que cela pourrait être factorisé par un include ? (https://jekyllrb.com/docs/includes/#passing-parameters-to-includes)